docs/snippets/add-or-modify-states.md

```diff
@@ -0,0 +1,30 @@
+# Add or modify states
+
+> This is a **Developer level** doc. If you are unfamiliar with code and resolving potential conflicts, select a [WooExpert or Developer](https://woocommerce.com/customizations/) for assistance. We are unable to provide support for customizations under our [Support Policy](http://www.woocommerce.com/support-policy/).
```

Comment:
Summarizing some internal discussion: we can drop these *"This is a **Developer level** doc"* warnings. Since this documentation targets developers, it is unnecessary.

Reply:
Removing this note based on p1692723749470709/1692723559.960259-slack-C0516GKR69W